kadi/commands/commands.py

```diff
@@ -135,8 +135,8 @@ def get_cmds_from_backstop(backstop, remove_starcat=False):
 
     n_bs = len(bs)
     out = {}
-    # Set idx to max (2**16 -1) so it does not match any real idx
-    out['idx'] = np.full(n_bs, fill_value=65535, dtype=np.uint16)
+    # Set idx to max (2**32 -1) so it does not match any real idx
+    out['idx'] = np.full(n_bs, fill_value=4294967295, dtype=np.uint32)
```

Comment:
Thanks Tom!  Can you talk a bit more about how this fill value is used?   Trying to make sure I'm not missing a subtlety in my review.  I'm guessing the error on "no id" that we started to see came up because a command in the archive and a command or commands from the backstop ended up with the same dummy id?

Reply:
It basically relates to the change below in `states.py`.  If the `idx` is not found in `REV_PARS_DICT` then it uses `cmd['params']` instead, which will always exist for commands read from backstop.

Reply:
OK. I suppose the problem there is that I don't recall what REV_PARS_DICT is supposed to be etc and if that try/except is clever optimization or required logic etc.  Will puzzle for a while.